Calendar/CalLogHelpers/MeetingSummaryFunctions.ps1

```diff
@@ -0,0 +1,105 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# Build-MeetingSummary.ps1
+# This script is used to support the Get-CalendarDiagnosticObjectsSummary.ps1 script.
```

Comment:
Since you removed them everywhere else, let's do it here as well.

Reply:
Done.